File: polocloud-suite/src/main/java/dev/httpmarco/polocloud/suite/commands/impl/ShutdownCommand.java
package dev.httpmarco.polocloud.suite.commands.impl;

import dev.httpmarco.polocloud.suite.PolocloudShutdownHandler;

public final class ShutdownCommand extends dev.httpmarco.polocloud.suite.commands.Command {

    public ShutdownCommand() {
        super("shutdown", "Shutdown this local suite", "stop");

        defaultExecution(commandContext -> PolocloudShutdownHandler.shutdown());
    }
}

File: agent/src/main/java/dev/httpmarco/polocloud/agent/runtime/local/terminal/commands/impl/ShutdownCommand.kt
package dev.httpmarco.polocloud.agent.runtime.local.terminal.commands.impl

import dev.httpmarco.polocloud.agent.exitPolocloud
import dev.httpmarco.polocloud.agent.i18n
import dev.httpmarco.polocloud.agent.runtime.local.terminal.commands.Command

class ShutdownCommand : Command(i18n.get("agent.terminal.command.shutdown.name"), i18n.get("agent.terminal.command.shutdown.description"), i18n.get("agent.terminal.command.shutdown.alias.1")) {

    init {
        defaultExecution {
            exitPolocloud()
        }
    }

}